templates/vue/README.md

```diff
@@ -38,8 +38,6 @@ You need to adjust your git config so the automatic linting process works correc
 git config core.hooksPath .githooks
 ```
 
-## Testing Setup
-
```

Comment:
Should we not keep a heading for this section? Otherwise it will just fall under the previous "Git Config" heading, which is not accurate.